com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.PostProcess.cs

```diff
@@ -377,8 +377,10 @@ void BeginPostProcessFrame(CommandBuffer cmd, HDCamera camera, HDRenderPipeline
                     }
                 }
 
+                // If the history was reset in the previous frame, then the history buffers were actually rendered with a neutral EV100 exposure multiplier
+                var prevExposureTexture = camera.didResetPostProcessingHistoryInLastFrame ? m_EmptyExposureTexture : GetPreviousExposureTexture(camera);
```

Comment:
Actually I changed my mind, I think is cleaner and simpler if we do the branch logic  inside GetPreviousExposureTexture itself, in case other user code grab this previous texture

Reply:
Indeed!